Expensify/react-native-live-markdown
# fix: improve performance of RN markdown parsing on large text

### Details


This PR includes minor improvement on `updateTreeElementRefs` function:
  + Remove the block to add the children of every child nodes to the stack. 
  + Query all elements with `data-id` attribute beforehand to improve performance

### Related Issues


https://github.com/Expensify/App/issues/46766

### Manual Tests


1. Paste this text file (over 400k characters) 
[default.txt](https://github.com/user-attachments/files/17113641/default.txt)

2. Observe the performance of the app's input

### Linked PRs


Pending App PR
